engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/StorageSystemDataMotionStrategy.java

```diff
@@ -2628,13 +2710,7 @@ private Map<String, String> getSnapshotDetails(SnapshotInfo snapshotInfo) {
 
         long snapshotId = snapshotInfo.getId();
 
-        // if the snapshot required a temporary volume be created check if the UUID is set so we can
-        // retrieve the temporary volume's path to use during remote copy
-        List<SnapshotDetailsVO> storedDetails = _snapshotDetailsDao.findDetails(snapshotInfo.getId(), "TemporaryVolumeCopyPath");
-        if (storedDetails != null && storedDetails.size() > 0) {
```

Comment:
any reason why we had to remove the first if condition ?

Reply:
This was a way to handle temporary volume creation in the original version of the Primera/Pure code committed in 4.19.0.0, but the code was refactored to not require this method of storing and later retrieving this temporary detail field.